firmware/console/binary/tunerstudio_outputs.h

```diff
@@ -95,7 +95,7 @@ typedef struct {
 
 	// air flow/mass measurment
 	scaled_voltage massAirFlowVoltage; // 26
-	scaled_channel<uint16_t, 100> massAirFlow; // 28
+	scaled_channel<uint16_t, 10> massAirFlow; // 28
```

Comment:
can we please extract this "10" magic constant into rusefi...txt for future reuse by console?

Reply:
let's see if PR auto-generate for builds works!

Reply:
ooh, it did!